jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/resources/org/jbpm/workbench/cm/client/resources/static/case-mgmt.less

```diff
@@ -0,0 +1,177 @@
+@import "/org/uberfire/client/views/static/uberfire-variables.less";
+@import "kie-variables.less";
+
+.kie-container--page {
+  margin-top: @pf-spacer-lg;
+}
+
+// override parts of patternfly horizontal nav and utility nav
+@navbar-pf-navbar-primary-bg-color-stop: @navbar-pf-bg-color; // #383F43
+@navbar-pf-navbar-utility-border-color: @navbar-pf-bg-color;
+@navbar-pf-navbar-utility-color: @color-pf-white;
+@navbar-pf-navbar-utility-hover-bg-color: @navbar-pf-bg-color;
+
+.navbar-toggle,
+.navbar-pf .navbar-utility {
+  top: 13px; //adjust top to vertically center the smaller item in the large masthead
+  .open > a,
+  .open > a:hover {
+    background-color: @navbar-pf-bg-color;
+  }
+}
+
+.kie-page__title {
+  font-size: @font-size-h1;
+  margin-bottom: @pf-spacer-lg;
+}
+
+// remove default space above and below breadcrumb
+.breadcrumb {
+  padding-bottom: 0;
+  padding-top: 0;
+}
+
+//remove default border and shadow from toolbar
+.kie-toolbar {
+  border-bottom: none;
+  box-shadow: none;
+}
+
+// title and subtitle in a toolbar
+.kie-toolbar__title {
```

Comment:
Can you add a padding in here, otherwise, the text is quite together with the owner string.
`padding-right: 5px;`